test/parallel/test-timers-ordering.js

```diff
@@ -23,11 +23,11 @@
 require('../common');
 const assert = require('assert');
 
-const Timer = process.binding('timer_wrap').Timer;
 const N = 30;
 
 let last_i = 0;
-let last_ts = 0;
+const [seconds, nanoseconds] = process.hrtime();
+let last_ts = (seconds * 1e3 + nanoseconds * 1e-6) | 0;
```

Comment:
Not sound.  `process.hrtime()` returns the number of nanoseconds since an unspecified point in the past.  If that point is the UNIX epoch then this stops working sometime next year because `seconds * 1e3 >= 2**32`.

Reply:
Ok, sounds like it's best to expose `getLibuvNow()` from `internal/timers` and just use that then.